release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -243,6 +243,18 @@ no operators found with name <name>.<version> in channel <name> of package <name
 
 === Operator development
 
+[id="ocp-4.8-sandboxed-containers"]
+=== {sandboxed-containers-first}
+
+[id="ocp-4.8-sandboxed-containers-tp"]
+==== {sandboxed-containers-first} support on {product-title} (Technology Preview)
+
+{sandboxed-containers-first} 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. {sandboxed-containers-first} enables users to choose Kata Containers as an additional runtime to provide additional isolation for their workloads. The {sandboxed-containers-operator} enables users to perform tasks such as installation, removal, updates, and status monitoring of Kata Containers.
```

Comment:
I am unclear how the operator allows status monitoring of Kata Containers.  I know of no specific feature for this.

Reply:
I don't think it means `status` literally, but rather status (for example, if you describe KataConfig you could see the progress.

Reply:
> The OpenShift sandboxed containers Operator enables users to perform tasks such as installation, removal, updates, and monitoring status using the `KataConfig` custom resource.

Reply:
Suggestion: The OpenShift sandboxed containers Operator enables users to perform tasks such as installation, removal, updates, and monitoring installation state visible when describing the KataConfig custom resource.